util/rlp/fuzz/build.rs

```diff
@@ -0,0 +1,23 @@
+extern crate rustc_version;
+use rustc_version::{version, version_meta, Channel, Version};
+
+fn main() {
+    assert!(version().unwrap().major >= 1);
+
+    match version_meta().unwrap().channel {
+        Channel::Stable => {
+            println!("cargo:rustc-cfg=RUSTC_IS_STABLE");
+        }
+        Channel::Beta => {
+            println!("cargo:rustc-cfg=RUSTC_IS_BETA");
+        }
+        Channel::Nightly => {
+            println!("cargo:rustc-cfg=RUSTC_IS_NIGHTLY");
+            println!("cargo:rustc-cfg=feature=\"nightly\"")
```

Comment:
what's the purpose of all the `RUSTC_IS_*`? can't you just `if let Channel::Nightly = version_meta().unwrap() { /* set feature flag */ }`